diff --git a/node_modules/expo-modules-core/ExpoModulesCore.podspec b/node_modules/expo-modules-core/ExpoModulesCore.podspec
index 136ea08..75febc9 100644
--- a/node_modules/expo-modules-core/ExpoModulesCore.podspec
+++ b/node_modules/expo-modules-core/ExpoModulesCore.podspec
@@ -16,6 +16,9 @@ fabric_compiler_flags = '-DRN_FABRIC_ENABLED -DRCT_NEW_ARCH_ENABLED'
 folly_version = '2022.05.16.00'
 folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -DFOLLY_CFG_NO_COROUTINES=1 -Wno-comma -Wno-shorten-64-to-32'
 
+expo_no_apns = ENV['EXPO_NO_APNS'] == '1'
+expo_no_apns_compiler_flags = '-DEXPO_NO_APNS'
+
 Pod::Spec.new do |s|
   s.name           = 'ExpoModulesCore'
   s.version        = package['version']
@@ -37,6 +40,7 @@ Pod::Spec.new do |s|
   header_search_paths = [
   ]
 
+ 
   # Swift/Objective-C compatibility
   s.pod_target_xcconfig = {
     'USE_HEADERMAP' => 'YES',
@@ -44,7 +48,9 @@ Pod::Spec.new do |s|
     'CLANG_CXX_LANGUAGE_STANDARD' => 'c++20',
     'SWIFT_COMPILATION_MODE' => 'wholemodule',
     'HEADER_SEARCH_PATHS' => header_search_paths.join(' '),
-    'OTHER_SWIFT_FLAGS' => "$(inherited) #{fabric_enabled ? fabric_compiler_flags : ''}"
+    'OTHER_CFLAGS' => "$(inherited) #{expo_no_apns ? expo_no_apns_compiler_flags : ''}",
+    'OTHER_CPLUSPLUSFLAGS' => "$(inherited) #{expo_no_apns ? expo_no_apns_compiler_flags : ''}",
+    'OTHER_SWIFT_FLAGS' => "$(inherited) #{expo_no_apns ? expo_no_apns_compiler_flags : ''} #{fabric_enabled ? fabric_compiler_flags : ''}"
   }
   user_header_search_paths = [
     '"${PODS_CONFIGURATION_BUILD_DIR}/ExpoModulesCore/Swift Compatibility Header"',
diff --git a/node_modules/expo-modules-core/ios/AppDelegates/EXLegacyAppDelegateWrapper.m b/node_modules/expo-modules-core/ios/AppDelegates/EXLegacyAppDelegateWrapper.m
index e0acffd..12af984 100644
--- a/node_modules/expo-modules-core/ios/AppDelegates/EXLegacyAppDelegateWrapper.m
+++ b/node_modules/expo-modules-core/ios/AppDelegates/EXLegacyAppDelegateWrapper.m
@@ -169,6 +169,7 @@ static dispatch_once_t onceToken;
 
 #pragma mark - Notifications
 
+#if !EXPO_NO_APNS
 - (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)token
 {
   SEL selector = @selector(application:didRegisterForRemoteNotificationsWithDeviceToken:);
@@ -221,6 +222,7 @@ static dispatch_once_t onceToken;
     }
   }
 }
+#endif
 
 #pragma mark - Subcontractors
 
diff --git a/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift b/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift
index 8ba09a7..f3c672e 100644
--- a/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift
+++ b/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift
@@ -103,6 +103,7 @@ open class ExpoAppDelegate: UIResponder, UIApplicationDelegate {
 
   // MARK: - Handling Remote Notification Registration
 
+#if !EXPO_NO_APNS
   open func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
     subscribers.forEach { $0.application?(application, didRegisterForRemoteNotificationsWithDeviceToken: deviceToken) }
   }
@@ -153,6 +154,7 @@ open class ExpoAppDelegate: UIResponder, UIApplicationDelegate {
       }
     }
   }
+#endif
 
   // MARK: - Continuing User Activity and Handling Quick Actions
 
