diff --git a/node_modules/expo-av/ios/EXAV/EXAVPlayerData.m b/node_modules/expo-av/ios/EXAV/EXAVPlayerData.m
index eef6690..48afd01 100644
--- a/node_modules/expo-av/ios/EXAV/EXAVPlayerData.m
+++ b/node_modules/expo-av/ios/EXAV/EXAVPlayerData.m
@@ -94,7 +94,16 @@ NSString *const EXAVPlayerDataObserverMetadataKeyPath = @"timedMetadata";
   
     _player = nil;
   
-    _url = [NSURL URLWithString:[source objectForKey:EXAVPlayerDataStatusURIKeyPath]];
+    // Fix for an issue where the URIs of built-in assets do not have the file://
+    // prefix after an OTA update is launched.
+    NSString *urlString = [source objectForKey:EXAVPlayerDataStatusURIKeyPath];
+    NSArray *urlParts = [urlString componentsSeparatedByString:@"://"];
+    if ([urlParts count] != 2) {
+      // there is no protocol prefix, so assume the file:// prefix
+      urlString = [@"file://" stringByAppendingString:urlString];
+    }
+    _url = [NSURL URLWithString:urlString];
+    
     _headers = [self validatedRequestHeaders:source[EXAVPlayerDataStatusHeadersKeyPath]];
   
     _timeObserver = nil;
