diff --git a/node_modules/react-native/Libraries/Components/Pressable/Pressable.js b/node_modules/react-native/Libraries/Components/Pressable/Pressable.js
index 96fe7d1..bccccfa 100644
--- a/node_modules/react-native/Libraries/Components/Pressable/Pressable.js
+++ b/node_modules/react-native/Libraries/Components/Pressable/Pressable.js
@@ -393,12 +393,12 @@ function Pressable(props: Props, forwardedRef): React.Node {
         }
       }
       // $FlowFixMe[prop-missing]
-      if (focused && evt.eventType === 'select') {
+      if (evt.eventType === 'select') {
         // $FlowFixMe[incompatible-exact]
         onPress && onPress(evt);
       }
       // $FlowFixMe[prop-missing]
-      if (focused && evt.eventType === 'longSelect') {
+      if (evt.eventType === 'longSelect') {
         // $FlowFixMe[incompatible-exact]
         onLongPress && onLongPress(evt);
       }
